instana-java-sdk/src/main/java/com/instana/sdk/annotation/Span.java

```diff
@@ -46,15 +46,19 @@
    *         {@link Object#toString()} method is invoked on all arguments. If any such invocation triggers an exception,
    *         the arguments for this methods are not stored. It is the responsibility of the user to assure that these
    *         methods do not imply any side-effects to the application.
+   * @deprecated use annotation {@code @TagParam }
    */
+  @Deprecated
   boolean captureArguments() default false;
 
   /**
    * @return Indicates that the return value of this method should be captured when it is exited. Doing so, the
    *         {@link Object#toString()} method is invoked on the return value. If this invocation triggers an exception,
    *         the return value is not stored. It is the responsibility of the user to assure that the invocation does not
    *         imply any unwanted side-effect to the application.
+   * @deprecated use annotation {@code @TagReturn }
```

Comment:
add ... "which gives you full control over the names of the tags to be created"

Reply:
done